### ISSUE TITLE
Create workplace facility and workplace function

### ISSUE THREAD

**Issue by dwelch2101** [ufbmi/OMRSE#18]
Needed for the synthetic ecosystem work on MIDAS

Possible definition of workplace function: "function adhering in a facility that manifests by the facility participating in carrying out the work of the organization"

**Comment by aellenhicks** [comment]
'workplace function' created - OMRSE_00000079
'workplace facility' created - OMRSE_00000078
Definitions and complete class restrictions still need to be added.

**Comment by aellenhicks** [comment]
Comments and questions on workplace function:
* Facilities are architectural structures, but not every workplace is in a building, e.g., boats and forests can be work places.
* In the case of workplace facilities, is it always the case that the facility participates in the carrying out of work?  Can it be the location without being a participant in the process?  (This gets into the murky territory of identity criteria for processes, but I think we should rephrase this to avoid talk of participation.)

**Comment by hoganwr** [comment]
For the Synthia data, we need only the workplaces in buildings, and that's
exactly what the proposed definition covers.  We have the same thing with
school facility and hospital facility right now.  Doesn't account for
homeschool or M.A.S.H units, but we really didn't need it to.

I am in line with rephrasing to avoid participation, but then we need to
fix school facility and hospital facility as well.  We could just use the
'occurs in' relation, or depending on order, its inverse 'site of'.

Bill

**Comment by aellenhicks** [comment]
How about the following? 
"a function inhering in a material entity that is realized by that material entity being the site where the work of some organization is carried out."

**Comment by hoganwr** [comment]
Looks good to me!
